pkgs/development/python-modules/tiledb/default.nix

```diff
@@ -0,0 +1,26 @@
+{  pkgs ? import ./pkgs.nix  }:
+with pkgs;
+let
+  python = python36;
+ 
+  tiledb  = import ./tiledb.nix { inherit pkgs; };
```

Comment:
You cannot do this for nixpkgs mainline. You need to get the package dependencies as a function.

See how other nixpkgs python modules are structured. Look at `numpy/default.nix`.

Reply:
ok